meilisearch-test-macro/src/lib.rs

```diff
@@ -169,9 +169,7 @@ pub fn meilisearch_test(params: TokenStream, input: TokenStream) -> TokenStream
 
         outer_fn.sig.inputs.clear();
         outer_fn.sig.asyncness = inner_fn.sig.asyncness.clone();
-        outer_fn
-            .attrs
-            .push(parse_quote!(#[futures_await_test::async_test]));
+        outer_fn.attrs.push(parse_quote!(#[tokio::test]));
```

Comment:
Oh, what is tokio test?

Reply:
Its the library used to handle async testing. We had to change it to be compatible with the new version of Mockito